.github/workflows/rebase.yml

```
@@ -8,12 +8,17 @@ jobs:
     if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '@jellyfin-bot rebase') && github.event.comment.author_association == 'MEMBER'
     runs-on: ubuntu-latest
     steps:
+      - name: Notify as seen
+        uses: peter-evans/create-or-update-comment@v1
```

```suggestion
        uses: peter-evans/create-or-update-comment@v1.4.4
```

Per marketplace **Use latest button**: https://github.com/marketplace/actions/create-or-update-comment